src/index.js

```diff
@@ -8,7 +8,7 @@ const prompts = require("prompts");
 const validateName = require("validate-npm-package-name");
 
 /**
- * @typedef {() => void} StepAction
+ * @typedef {function():void} action
```

Comment:
```suggestion
 * @typedef {function(): void} StepAction
```
nit: PascalCase naming convention and spacing